### ISSUE TITLE
Allow alias to be used for long job names

### ISSUE THREAD

**Issue by ghopson** [radarsh/terminator#11]
Supply an alias on the job name separated by a | or similar and use that if available on the display

E.g. -j "software-app-some-service/job/develop|SomeService software-app-other-service/job/develop|Banana software-app-long-service/job/develop"

(Note last one has no alias)

**Comment by radarsh** [comment]
Good idea. Considering that the command line is getting longer and longer I am thinking of providing an option for specifying it via a config file as an alternative.

**Comment by ghopson** [comment]
Do both :-)
